pkg/frr/cli.go

```diff
@@ -0,0 +1,243 @@
+package frr
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+type Cli struct {
+	binaryPath string
+}
+
+func NewCli() *Cli {
+	return &Cli{
+		binaryPath: "/usr/bin/vtysh",
+	}
+}
+
+// getVRF returns either the provided vrf from the function params or if its empty.
+// the constant string "all" as the vrf name.
+// if it returns it also provides feedback if the vrf parameter was empty or not as
+// the second return value.
+
+func getVRF(vrf string) (string, bool) {
+	if vrf == "" {
+		return "all", true
+	}
+	return vrf, false
+}
```

Comment:
Tried to infer from the usage what the return values actually mean to maybe make it a bit easier to read.
Required rename of the function of course

```suggestion
// getVRFInfo returns either the provided vrf from the function params - or if it's empty -
// the constant string "all" as the vrf name as well as whether it received an empty input
func getVRFInfo(vrf string) (name string, isMulti bool) {
	if vrf == "" {
		return "all", true
	}
	return vrf, false
}
```